cli.js

```diff
@@ -59,7 +59,9 @@ if (cli.input.length > 0) {
 	fetch(cli.input[0]).then(val => {
 		val = val.map(x => skinTone(x, skinNumber));
 		console.log(val.join('  '));
-		clipboardy.writeSync(val[0]);
+		if (cli.flags.copy) {
```

Comment:
I know this isn't related to the skin tones but I noticed that the copy flag wasn't actually being checked and the first emoji would always be copied to clipboard when an argument is given regardless of the flag.